### PR TITLE
UVF time support added/tested

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changelog {#Changelog}
 
 * [#145](https://github.com/BlueBrain/Livre/pull/145):
   Added multisampling per pixel and sparsity parameter for memory datasource
+* [#145](https://github.com/BlueBrain/Livre/pull/145):
+  Added multisampling per pixel and sparsity parameter for memory datasource
```
Comment (eile) [inline doc/Changelog.md]: dup?

**Comment by eile** [comment]
+1

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
RRRRhh ... Jenkins cant event checkout !!!

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
@eile  Collage is "back in black". The source of the issue is known somehow ?

**Comment by eile** [comment]
dataStreamArchive... that's something new which hasn't changed in eons. :(

Looking into it.

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Pull request merged**